src/main/java/org/takes/facets/auth/social/PsTwitter.java

```diff
@@ -136,19 +184,4 @@ private String token()
             .as(JsonResponse.class)
             .json().readObject().getString("access_token");
     }
-
-    /**
-     * Make identity from JSON object.
-     * @param json JSON received from Twitter
-     * @return Identity found
-     */
-    private static Identity parse(final JsonObject json) {
```

Comment:
Two reasons:
1) it was a static method http://www.yegor256.com/2015/02/20/utility-classes-vs-functional-programming.html
2) it was used only in one method